### ISSUE TITLE
Add configurable threshold for urgent tasks

### ISSUE THREAD

**Issue by jadoro** [pimutils/todoman#205]
`todoman`'s current (`v2.1.0`) behavior of interpreting all `VTODO:PRIORITY` fields greater than zero as "urgent" leads to problems when using the same files in parallel with tools that implement non-binary priority: Task with lowest priority (e.g. `1`) will become "urgent" in todoman.

My proposal would be an adjustable threshold for urgent tasks, settable via the main config file. Tasks with priority below this threshold (e.g. `<5`) are not urgent, tasks with priority above are. New urgent tasks would be created with the given threshold as priority.
With a default setting of `1`, this additional option would not break backwards compatibility for existing installations.

**Comment by untitaker** [comment]
`1` is the highest priority, `9` the lowest. https://tools.ietf.org/html/rfc5545#section-3.8.1.9

**Comment by hobarrera** [comment]
Sounds good to me. I'd rather expose this as high/medium/low, instead of the actual raw values though.

**Comment by jadoro** [comment]
> 1 is the highest priority, 9 the lowest.

You're right, my bad.

The issue that `todoman` recognizes "low priority tasks" (`VTODO:PRIORITY = 9`, I just double-checked that) as urgent holds nevertheless.

> I'd rather expose this as high/medium/low, instead of the actual raw values though.

That would surely also be fine. I guess that the current implicit setting is that any value != 0 corresponds to "urgent"?

**Comment by hobarrera** [comment]
This is already fixed in master (`--urgent` is replaced by `--priority`). (Sorry, I though that was a pending issue).

It'll be included in the v3.0.0 release, which TBH, is probably overdue.
